docs/book/v3/reference/cli-tooling.md

```diff
@@ -73,7 +73,10 @@ Commands supported include:
 - **`handler:create [options] <handler>`**: Create a request handler named after
   `<handler>`. By default, the command will also generate a factory, register
   both with the application container, and, if a template renderer is
-  discovered, generate a template in an appropriate location.
+  discovered, generate a template in an appropriate location. **Note:**
+  `<handler>` will either need to be quoted or the backslashes escaped.
+  **Example:** `handler:create ModuleName\\Handler\\IndexHandler` or
```

Comment:
The examples should be turned over, then they would match the description.

Reply:
Good eye. :)